src/test/scala/uk/gov/nationalarchives/tdr/api/service/ValidateFileMetadataServiceSpec.scala

```diff
@@ -153,17 +158,19 @@ class ValidateFileMetadataServiceSpec extends AnyFlatSpec with MockitoSugar with
     val service = testSetUp.service
     val response = service.validateAdditionalMetadata(fileIds, Set("ClosureType", "description")).futureValue
 
-    response.size shouldBe 4
+    val convertedResponse = convertFilestatusRowToAddFileStatusInput(response)
+
+    convertedResponse.size shouldBe 4
 
     verify(testSetUp.mockFileStatusRepository, times(1)).deleteFileStatus(fileIds, Set(ClosureMetadata, DescriptiveMetadata))
-    verify(testSetUp.mockFileStatusRepository, times(1)).addFileStatuses(response)
+    verify(testSetUp.mockFileStatusRepository, times(1)).addFileStatuses(any[List[AddFileStatusInput]])
```

Comment:
Ithink you can verify the input for this method as your "converted response" as you've done in the tests above

Reply:
Instead of passing in the `any[List[AddFileStatusInput]]` pass in the actual value derived from your response object, ie:

```
...
  val fileStatusResponseConvertedToAddFileStatusInput = convertFileStatusRowToAddFileStatusInput(response)
  ...
  verify(testSetUp.mockFileStatusRepository, times(1)).addFileStatuses(fileStatusResponseConvertedToAddFileStatusInput)
...
```